scripts/func_defs.py

```diff
@@ -129,7 +129,7 @@ def get_fastq_name(full_name):
     elif(find_pe_inx>=0):
      output=full_name[:find_pe_inx]
     else:
-     print("Sth went wrong")
+     print("Unable to infer Sample fastq name from Snakemake-generated string.")
```

Comment:
maybe rather put something like this:
`print("Unable to infer Sample fastq name, cannot find trimming flag in file name. \nHave the files been adapter and quality trimmed?")`

Reply:
Actually, this procedure has always annoyed me, because it doesn't do the right thing.

If there's an error we should bail out.  Printing a message doesn't help.  This procedure won't stop execution of the pipeline and will just delay the fatal error.

The right thing to do here is to fail.  (Or to replace this procedure completely; ISTR that we have at least one variant of this procedure in some other file.)

But none of this matters for this set of changes.  Changing the message is an improvement in any case, even though it is "putting lipstick on a pigx" :)

Reply:
Agreed, I was just scrolling through this file at some point and figured that _any_ information was better than "Sth went wrong"

Reply:
These suggestions were implemented in the latest commit (just now)